airbyte-scheduler/scheduler-persistence/src/test/java/io/airbyte/scheduler/persistence/DefaultJobCreatorTest.java

```diff
@@ -337,7 +341,8 @@ void testCreateResetConnectionJob() throws IOException {
         .withDestinationDockerImage(DESTINATION_IMAGE_NAME)
         .withConfiguredAirbyteCatalog(expectedCatalog)
         .withOperationSequence(List.of(STANDARD_SYNC_OPERATION))
-        .withResourceRequirements(workerResourceRequirements);
+        .withResourceRequirements(workerResourceRequirements)
+        .withResetSourceConfiguration(new ResetSourceConfiguration().withStreamDescriptors(List.of(STREAM_DESCRIPTOR1, STREAM_DESCRIPTOR2)));
```

Comment:
For `ResetSourceConfiguration` would it make sense to call the field `streamsToReset` instead of `streamDescriptors`? Seems like the former is the variable name we are using elsewhere. Looks like this happened in an earlier PR, but just seeing it now.

Reply:
I don't feel too strongly here - I think `streamsToReset` makes sense, so I can make that change

Reply:
@benmoriceau I just pushed the change to rename it